recipes/fs.sshfs/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "fs.sshfs" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e9f218a382c8a82c9f639d1c0e3a622ed005fed03923966621bafbfe0da12f4
+
+build:
+  skip: true  # [win]
+  noarch: python
```

Comment:
Can't be `noarch` if it needs selectors unfortunately. So this will need to be dropped.